.github/workflows/main.yaml

```
@@ -1,20 +1,30 @@
 name: build
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+    - 'packages/**'
+    - '!packages/*/example/**'
+    - 'examples/**'
+  pull_request:
+    paths:
+    - 'packages/**'
+    - '!packages/*/example/**'
```

```suggestion
    - 'performance_tests/**'
```

We don't need this anymore. Plus we forgot including this folder. :)

Protip: you can just click in the PR "accept suggestion" to speed up the process. ;)

---

@netantho this is still wrong. Btw. if you would have use the suggestions and accepted them, it would work out. :P

---

@tenhobi I'm very sorry, I read too fast, got confused with another comment. 🤦‍♂  Thank you very much for checking!